odbc/src/api/col_attr_describe_tests.rs

```diff
@@ -566,7 +566,7 @@ mod unit {
                 .unwrap();
             assert_eq!(errors.len(), 1);
             assert_eq!(
-                "[MongoDB][API] Driver not capable".to_string(),
+                "[MongoDB][API] Invalid field descriptor value 4".to_string(),
```

Comment:
4 ?

Reply:
This is the case where the value passed in to descriptor doesn't map to any values in the enum (as opposed to a value in the enum that we dont support), and I thought it would be useful to include the value in the error message. This is a call that SSIS makes:
````
1ee0-1414	ENTER SQLColAttributeW 
		SQLHSTMT            0x00000223EE458400
		SQLSMALLINT                  1 
		SQLSMALLINT                  4 <unknown>
		SQLPOINTER         0x00000223EBAE6A40
		SQLSMALLINT               4096 
		SQLSMALLINT *       0x000000322297A868
		SQLPOINTER          0x000000322297A808
```

Reply:
oh, right because we don't have a name for it, makes sense, sorry 👍